clash-cores/src/Clash/Cores/Xilinx/VIO/Internal/BlackBoxes.hs

```diff
@@ -85,9 +85,11 @@ vioProbeTF =
     (
 --      0 :       -- KnownDomain dom
 --      1 :       -- VIO dom a o
-      2 :       -- initialOutputProbeValues
-      3 :       -- Clock dom
-      [4,5..]   -- input probes
+      2 :       -- input probe names
+      3 :       -- output probe names
+      4 :       -- initialOutputProbeValues
+      5 :       -- Clock dom
+      [6,7..]   -- input probes
```

Comment:
We can use `infinite-list` now. For an example, see: https://github.com/clash-lang/clash-compiler/blob/bda8bcca55ec3ff3427ae62e9b7b36fc47f3bff9/clash-cores/src/Clash/Cores/Xilinx/Xpm/Cdc/Gray/Internal.hs#L47-L60.